R/mergelist.R

```diff
@@ -0,0 +1,335 @@
+cbindlist = function(l, copy=TRUE) {
+  ans = .Call(Ccbindlist, l, copy)
+  if (anyDuplicated(names(ans))) { ## invalidate key and index
+    setattr(ans, "sorted", NULL)
```

Comment:
why this non-standard way to reset key/index? shouldn't we re-use abstractions here rather than copy implementation details in multiple places?